Cargo.toml

```
@@ -1,25 +1,5 @@
-[package]
-name = "k8s-gateway-api"
-version = "0.4.0"
-edition = "2021"
-license = "Apache-2.0"
-repository = "https://github.com/linkerd/k8s-gateway-api"
-rust-version = "1.60"
-keywords = ["kubernetes", "gateway"]
-description = "Rust bindings for the Kubenetes Gateway API"
-
-[dependencies]
-kube = { version = "0.73", default-features = false, features = ["derive"] }
-k8s-openapi = { version = "0.15", features = ["schemars"] }
-schemars = { version = "0.8", features = ["derive"] }
-serde = { version = "1", features = ["derive"] }
-serde_json = "1"
-
-[dev-dependencies]
-k8s-openapi = { version = "0.15", default-features = false, features = ["v1_24"] }
-
-[package.metadata.docs.rs]
-rustdoc-args = ["--cfg", "docsrs"]
-features = [
-    "k8s-openapi/v1_24",
-]
+[workspace]
+members = [
+    "k8s-gateway-api",
+    "integration",
+]
```

I think in this case we could avoid moving all of the k8s-gateway-api stuff into its own directly and instead do 
```suggestion
[workspace]
members = [
    ".",
    "integration",
]
```

leaving the rest of the original crate definition here.